index.html

```diff
@@ -94,12 +94,12 @@ <h3>Portfolio Template</h3>
                   Visit Site <span>&rarr;</span>
                 </a>
                 <a href="https://github.com/nisarhassan12/portfolio" title="View Source Code" target="_blank">
-                  <img src="./images/github.svg" class="work__code" alt="GitHub">
+                  <img src="./images/github.svg" class="work__code" alt="Github link to source code">
                 </a>
               </div>
             </div>
             <div class="work__image-box">
-              <img src="./images/project-1.png" class="work__image" alt="Project 1" />
+              <img src="./images/project-1.png" class="work__image" alt="Image showcasing Portfolio Template" />
```

Comment:
It is always a good idea to avoid words like image and photo in the alt texts as they are redundant.

![image](https://user-images.githubusercontent.com/46004116/83319811-1bc76f00-a25b-11ea-8fc2-efa6a335c997.png)

Source: https://moz.com/learn/seo/alt-text
```suggestion
              <img src="./images/project-1.png" class="work__image" alt="Portfolio Template" />
```